core/application/react-query-adapter/billing-profile/client/use-delete-billing-profile-coworker.ts

```
@@ -0,0 +1,46 @@
+
+import {
+  UseMutationFacadeParams,
+  useMutationAdapter,
+} from "@/core/application/react-query-adapter/helpers/use-mutation-adapter";
+import { bootstrap } from "@/core/bootstrap";
+import { BillingProfileFacadePort } from "@/core/domain/billing-profile/input/billing-profile-facade-port";
+
+export function useDeleteBillingProfileCoworker({
+  pathParams,
+  options,
+}: UseMutationFacadeParams<BillingProfileFacadePort["deleteBillingProfileCoworker"], undefined, void, {}> = {}) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)